app/helpers/application_helper/button/pdf.rb

```diff
@@ -2,4 +2,8 @@ class ApplicationHelper::Button::Pdf < ApplicationHelper::Button::Basic
   def visible?
     PdfGenerator.available?
   end
+
+  def check_rbac?
+    %w(render_report_pdf chargeback_download_pdf compare_download_pdf).include?(self[:child_id]) ? role_allows?(:feature => self[:child_id]) : true
```

Comment:
@romanblanco, @PanSpagetka, @jzigmund : discussed with @h-kataria and @dclarizio and there really should be an RBAC feature for each button.

Please, add the missing features and add those features as ~~hidden~~ features.

Then remove this exception from the code.

Thx!